### PR TITLE
add bind and unbind with arguments methods

### PR DESCRIPTION
**Pull request by ntt188**
Motivation:

Bind and unbind with arguments methods are missing.

Related issues: #25 #28

### CODE REVIEW

**Comment by ntt188** [comment]
@vietj @cescoffier @Sammers21 Can you take a look at my PR please?

File: src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java [hunk 1]
```diff
@@ -26,11 +28,12 @@
   protected RabbitMQClient client;
   protected Channel channel;
   protected Vertx vertx;
+  protected RabbitMQManagementClient managementClient;
 
   @ClassRule
-  public static final GenericContainer rabbitmq = new FixedHostPortGenericContainer<>("rabbitmq:3.7")
+  public static final GenericContainer rabbitmq = new FixedHostPortGenericContainer<>("rabbitmq:3.7-management")
```
Comment (vietj) [inline src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java]: why is this change needed  ? more curious than questioning it

**Comment by vietj** [comment]
can you submit this PR for 3.9 branch please @ntt188 ? I'll review it after for 3.9 inclusion

File: src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java (hunk 1 above)
Comment (ntt188) [reply src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java]: I use management api to test bind and unbind methods.

Comment (cescoffier) [reply src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java]: Any reason to used a Fixed Port container? I know it was there before, but maybe a good time to change this.

**Comment by ntt188** [comment]
> can you submit this PR for 3.9 branch please @ntt188 ? I'll review it after for 3.9 inclusion
@vietj Done.

File: src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java (hunk 1 above)
Comment (ntt188) [reply src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java]: No, there is no reason to use fixed port container. I'll make changes to use random port.

Comment (ntt188) [reply src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java]: @cescoffier Fixed ports here are exposed ports. Mapped ports are already random.

**Comment by vietj** [comment]
thanks for the contribution - ported to master
